Tests/test_CTD.py

```diff
@@ -0,0 +1,124 @@
+# This code is part of the Biopython distribution and governed by its
```

Comment:
Please use the dual licensing header.

Reply:
Will change, didn't realize that there were two different versions and must have copied it from two different spots.